doc/source/user/basics.io.genfromtxt.rst

```diff
@@ -58,8 +58,8 @@ Quite often, a single character marks the separation between columns.  For
 example, comma-separated files (CSV) use a comma (``,``) or a semicolon
 (``;``) as delimiter::
 
-   >>> data = "1, 2, 3\n4, 5, 6"
-   >>> np.genfromtxt(BytesIO(data), delimiter=",")
+   >>> data = u"1, 2, 3\n4, 5, 6"
+   >>> np.genfromtxt(StringIO(data), delimiter=",")
```

Comment:
the delimiter should be an u string too

Reply:
hm though there are a lot, maybe it is unnecessary clutter

Reply:
The delimiter gets promoted to unicode in Python 2.7, so I think this is OK.
```
In [1]: from io import StringIO

In [2]: data = u"1, 2, 3\n4, 5, 6"

In [3]: import numpy as np

In [4]: np.genfromtxt(StringIO(data), delimiter=",")
Out[4]: 
array([[1., 2., 3.],
       [4., 5., 6.]])
```